sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/UIUtils.scala

```diff
@@ -57,4 +62,16 @@ private[ui] object UIUtils {
       query.exception.map(_ => "FAILED").getOrElse("FINISHED")
     }
   }
+
+  private val progressTimestampFormat = new ThreadLocal[SimpleDateFormat]() {
+    override def initialValue(): SimpleDateFormat = {
+      val format = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSS'Z'") // ISO8601
```

Comment:
Any particular reasons for using SimpleDateFormat instead of DateTimeFormatter? Just in case, would it possible to use `TimestampFormatter` here?

Reply:
This is to use the same formatter as https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ProgressReporter.scala#L91